examples/spring-boot/marketplace-app/src/main/java/com/example/vendasta/ShoeStore/utils/CommonUtils.java

```diff
@@ -0,0 +1,47 @@
+package com.example.vendasta.ShoeStore.utils;
+
+import java.time.Instant;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class CommonUtils {
```

Comment:
Generally having a generic `utils` package is a code smell. It leads to bloated packages and classes that have a whole bunch of unrelated functions grouped together which makes it harder to test.

Reply:
Okay, I get it. Should I remove the file?

Reply:
Class name changed to **ParserUtils**.